### PR TITLE
Change in --help commands of read and write

### PR DESCRIPTION
**Pull request by bgshri**
Solves https://github.com/hashicorp/vault/issues/16788

### CODE REVIEW

**Comment by aphorise** [comment]
@bgshri
1. Add `changelog/17011.txt` with something like the following:
    ```
    cli: CLI vault read / write examples updated in help pages.
    ```
2. Sign CLA

**Comment by bgshri** [comment]
@aphorise 
1. Added the change log

2. Already signed CLA but its not reflecting

**Comment by peteski22** [comment]
Hi @bgshri , thanks for your submission. Are you able to go through the process of signing the CLA again please? The error message in Github maybe implies that a different account was tried when signing? 

`bbhaya seems not to be a GitHub user.`

File: changelog/17011.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+cli: Example vault read / write updated in help page.
```
Comment (aphorise) [inline changelog/17011.txt:2]: Can you please change to something like:
`cli: Example 'help' page KV read / write docs improved.`

**Comment by aphorise** [comment]
I've gone ahead and created: #19064 - with the small changes that supersedes this PR.
 
Hey @bgshri - thanks for making the effort to raise this.

**Pull request closed without merging**